device/dell/x86_64-dellemc_s5248f_c3538-r0/DellEMC-S5248f-P-25G/gearbox_config.json

```diff
@@ -0,0 +1,38 @@
+{
+  "phys": [
+    {
+      "phy_id": 1,
+      "name": "sesto-1",
+      "address": "0x1000",
+      "lib_name": "libsai_phy_sesto-1.so",
```

Comment:
why do we need to specify the library name here. it seems unnecessary.

Reply:
The original requirement was to support the flexibility that each PHY could have their own so file. Please note that this is currently a placeholder and can change as required.